examples/react-spa/src/Recovery.tsx

```diff
@@ -42,9 +42,9 @@ export const Recovery = () => {
 
     sdk
       .updateRecoveryFlow({ flow: flow.id, updateRecoveryFlowBody: body })
-      .then(() => {
-        // we successfully submitted the login flow, so lets redirect to the dashboard
-        navigate("/", { replace: true })
+      .then(({ data: flow }) => {
+        // Form submission was successful, show the message to the user!
+        setFlow(flow)
```

Comment:
I think this only works partially since there are 2 parts:

1. Submit the email
2. Submit the code

On the second submit we need to navigate to the settings page on Recovery. With this we will be able to submit everything fine, but the redirect wouldn't kick in which means the user will be stuck on the recovery page.

Reply:
The user won't be stuck on the recovery page with this change, but actually ends up to the error page.

Have to do the same improvement to error code 422 handling as nextjs example got today.

Reply:
Yeah only when it errors, but what i mean is after a successful submission of the email (step 1) and code (step 2) the browser should navigate to the settings page.

Reply:
After (step 2) when verify code has been given successfully the response from the server is error code 422 which redirects to the settings page. Currently, it does not work because 422 does not handle path change. Just got it fixed and tested and it works.